src/app/components/turnos/gestor-agendas/operaciones-agenda/planificar-agenda.component.ts

```diff
@@ -253,6 +257,28 @@ export class PlanificarAgendaComponent implements OnInit, AfterViewInit {
         }
     }
 
+    seleccionarMultiprofesional() {
+        if (this.multiprofesional) {
+            if (this.noNominalizada) {
+                this.plex.info('warning', 'No se puede configurar como multiprofesional ya que la prestación seleccionada es no nominalizada').then(() => {
+                    this.multiprofesional = false;
+                });
+            } else {
+                this.noNominalizada = false;
```

Comment:
Si en la linea 262 this.noNominalizada es false, entonces en la linea 267 no es necesario volver a declararla como false ya que su valor no es modificado.

Reply:
arreglado!